framework/src/utils/SlepcSupport.C

```diff
@@ -19,11 +19,15 @@
 #include "EigenProblem.h"
 #include "FEProblemBase.h"
 #include "NonlinearEigenSystem.h"
-
+// libMesh
 #include "libmesh/petsc_vector.h"
 #include "libmesh/petsc_matrix.h"
 #include "libmesh/slepc_macro.h"
 #include "libmesh/auto_ptr.h"
+// PETSc
+#include "petscsnes.h"
+// SLEPc
```

Comment:
I do not think we need the comments myself. I think the include names speak for themselves. But I do not care too much about this

Reply:
Hmmm

Reply:
Hmmm